src/concurrency/genmc/helper.rs

```diff
@@ -95,3 +98,55 @@ impl AtomicWriteOrd {
         }
     }
 }
+
+impl AtomicFenceOrd {
+    pub(super) fn to_genmc(self) -> MemOrdering {
+        match self {
+            AtomicFenceOrd::Acquire => MemOrdering::Acquire,
+            AtomicFenceOrd::Release => MemOrdering::Release,
+            AtomicFenceOrd::AcqRel => MemOrdering::AcquireRelease,
+            AtomicFenceOrd::SeqCst => MemOrdering::SequentiallyConsistent,
+        }
+    }
+}
+
+impl AtomicRwOrd {
+    /// Split up the atomic success ordering of a read-modify-write operation into GenMC's representation.
+    /// Note that both returned orderings are currently identical, because this is what GenMC expects.
```

Comment:
If genmc expects a single ordering then it'd make more sense to just return a single ordering here.